src/Model/ModelManager.php

```diff
@@ -814,4 +820,30 @@ private function getValueFromType($value, Type $type, string $fieldType, Abstrac
 
         return (string) $type->convertToDatabaseValue($value, $platform);
     }
+
+    private function getFieldCustomType(string $fieldType): Type
+    {
+        return Type::getType($fieldType);
+    }
+
+    private function buildInnerIdentifier(string $prefix, string $field, $value, int $pos, QueryBuilder $qb, ClassMetadata $classMetadata): string
```

Comment:
There is a lot of parameters in this private method.

Is there a way to reduce the number ?
Maybe the parameterName can be an argument instead of passing the prefix and the pos.

Reply:
Initially wanted to make `addIdentifiersToQuery` a bit cleaner and remove chunk of code from foreach-loop but other hand I can revert this private method

Reply:
The method `addIdentifiersToQuery` is complex, so this private method has a lot of sens. 

```
$parameterName = sprintf('field_%s_%s_%d', $prefix, $name, $pos);
$this-> buildInnerIdentifier($field, $value, $paraterName, $qb, $classMetadata)
```
Was just a suggestion, not a blocking point.

Reply:
It will reduce a few args. Good idea. Will do it.